src/views/UrbitID/SetProxy.js

```diff
@@ -113,7 +129,30 @@ export default function SetProxy() {
     bind,
   } = useSetProxy(data.proxyType);
 
+  const setProxy = useCallback(async () => {
+    // setLoading(true);
+    console.log(newAddress);
+    if (newAddress === '') return;
+    try {
+      await setProxyAddress(getL2ProxyName(data.proxyType), newAddress);
+      getPendingTransactions(_point);
+      pop();
+    } catch (error) {
+      // setError(error);
```

Comment:
Do we still need the catch / finally? We an also address this in a follow-up PR

Reply:
Yes, let's do that when we tackle the new UI